lib/components/atoms/Popover/Popover.style.js

```diff
@@ -0,0 +1,5 @@
+import { css } from 'styled-components';
+
+export default css`
+  ${props => (props.inheritedStyles ? props.inheritedStyles : '')};
```

Comment:
Please do right bare minimum CSS like: Border Box, Small padding to all 4 side of the content.